setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = stips
 # version should be PEP440 compatible (https://www.python.org/dev/peps/pep-0440/)
-version = 1.0.7
+version = 1.0.8
```

Comment:
The version change should not happen here. It should happen in the prerelease PR #69

Reply:
Okay.